From b7af695d79820adf53e7d612120bda12ed2886e2 Mon Sep 17 00:00:00 2001
From: Marek Vasut <marex@denx.de>
Date: Tue, 9 May 2023 02:57:38 +0200
Subject: [PATCH 2/2] wayland: Fix callback data type

The LVGL 8.3.y changed the callback data type, update it accordingly.

Upstream-Status: Inappropriate [Upstream repo is archived]
Signed-off-by: Marek Vasut <marex@denx.de>
---
 src/drivers/wayland.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/src/drivers/wayland.c b/src/drivers/wayland.c
index bcebf4d..cfefa88 100644
--- a/src/drivers/wayland.c
+++ b/src/drivers/wayland.c
@@ -37,8 +37,7 @@ uint32_t custom_tick_get(void)
 	return time_ms;
 }
 
-
-static lv_wayland_display_close_f_t close_cb()
+static bool close_cb(lv_disp_t * disp)
 {
 }
 
-- 
2.39.2

